tests/test_server.py

```diff
@@ -66,3 +66,20 @@ def test_route_with_publisher_service_absent(mocker):
     }
     assert json.loads(response.get_data()) == output
     assert response.status_code == 500
+
+
+def test_ping(mocker):
+    """Test Ping."""
+    client = APP.test_client(mocker)
+
+    url = '/ping'
+
+    response = client.get(url)
+
+    output = {
+        "message": "Alive",
+        "status": "OK",
+        "version": VERSION
+    }
+    assert json.loads(response.get_data()) == output
```

Comment:
```suggestion
    assert response.get_json() == output
```

Reply:
Sure. I have also made the same change in other tests.

Currently the AI Service liveness/readiness uses the root `/` GET route, where it checks for the availability of publisher -- the rationale behind that was pretty much the same (as discussed in https://github.com/ManageIQ/aiops-data-collector/pull/46#issuecomment-510973827) , basically don't let the analysis go waste, hence first ensure that we have publisher available.

Based on what we decide in https://github.com/ManageIQ/aiops-data-collector/pull/46, I can make a change here to check for publisher availability as an initial configuration check, and not include it as part of the probe.